friends/src/App.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import axios from 'axios'
+import Friend from './Friend/Friend';
+import NewFriend from './NewFriend/NewFriend';
+import './App.scss';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      postRes: '',
+      friend: {
```

Comment:
Since the App component does not need to share the `friend` state with other child component except for the form `NewFriend` component, you can move `friend` state to that component instead, and manage them with onChange event handler. Then send a method to change the `friends` state down through props like:
```
  handleAddedFriend = friends => this.setState({friends})
```